oci/archive/oci_src.go

```diff
@@ -72,12 +81,10 @@ func (s *ociArchiveImageSource) Reference() types.ImageReference {
 }
 
 // Close removes resources associated with an initialized ImageSource, if any.
-// Close deletes the temporary directory at dst
 func (s *ociArchiveImageSource) Close() error {
```

Comment:
This needs to close the `Reader` if it is single-use = we have created it for this `ImageSource`, so that the temporary directory is deleted.

Reply:
fixed